superset/assets/javascripts/dashboard/components/Controls.jsx

```diff
@@ -36,11 +45,11 @@ class Controls extends React.PureComponent {
   }
   refresh() {
     // Force refresh all slices
-    this.props.dashboard.renderSlices(this.props.dashboard.sliceObjects, true);
+    this.props.renderSlices(true);
   }
   changeCss(css) {
     this.setState({ css });
-    this.props.dashboard.onChange();
+    this.props.onChange();
   }
   render() {
     const dashboard = this.props.dashboard;
```

Comment:
you could destructure all of the variables used in the calls to `this.props.xxx` below to simplify.